cmake/tribits/core/package_arch/TribitsGlobalMacros.cmake

```diff
@@ -2177,7 +2177,7 @@ macro(tribits_configure_enabled_packages)
         endif()
         if (NOT ${PACKAGE_NAME}_TRIBITS_PACKAGE_POSTPROCESS)
           tribits_report_invalid_tribits_usage(
```

Comment:
I think I will be able to adjust the logic to avoid this check if the target `${PACKAGE_NAME}::all_libs` is already defined.

I will be creating a PR that will duplicate this Kokkos using case (i.e. using a raw CMake build system for a TriBITS-compliant internal package that does not call any TrBITS functions).  This is one of the issues I will hit with that.

Reply:
Yeah, that would be good. I think we would rather define the target then calling `tribits_package_postprocess()`.